index.html

```diff
@@ -47,6 +47,212 @@ <h1>I’m Samuel Glad to see you!</h1>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="cardworks4">
+            <div class="snapshot">
+                <img src="images/4.png" alt="snapshot portfolio">
+            </div>
+            <div class="leftblock">
+                <div class="prim-txt">
+                    <h2>Multi-Post Stories</h2>
+                    <img src="images/Frame 2.png" alt="works">
+                </div>
+                <P>A daily selection of privately personalized reads; no accounts or sign-ups required.</P>
+                <div class="tag">
+                    <ul>
+                        <div class="html">
+                            <li>
+                                <a href="#">html</a>
+                            </li>
+                        </div>
+                        <div class="css">
+                            <li>
+                                <a href="#">css</a>
+                            </li>
+                        </div>
+                        <div class="js">
+                            <li>
+                                <a href="#">javascript</a>
+                            </li>
+                        </div>
+                        
+                    </ul>
+                </div>
+                <div class="action">
+                    <div class="button">
+                        <p>See Project</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about_myself">
+            <div class="pr-t">
+                <h1>About Myself</h1>
+            </div>
+            <div class="sec-t">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="soci-med">
+                <p>LET’S CONNECT</p>
+            </div>
+            <div class="socm-icons">
+                <ul class="icons">
+                    <li>
+                        <a href="#"><img src="images/Vector.svg" alt="twitter icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedn.svg" alt="linkedn icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/medium.svg" alt="medium icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/github.svg" alt="github icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/finger.svg" alt="finger icons"></a>
+                    </li>
+                </ul>
+            </div>
+            <div class="normalbutton">
+                <p>Get my resume</p>
+            </div>
```

Comment:
- [ ] Let's improve our current design to match Figma design by changing the `font-family`, adjust  the spacing properties (box-model) and add some interactions to our button as shown below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219575883-7cf13e32-4a42-4b93-9bd8-4616062c7520.png)|![image](https://user-images.githubusercontent.com/99715640/219575957-5c7003e4-6953-4f51-a2f5-aef4cb87aff7.png)|